spec/venue_spec.rb

```diff
@@ -43,13 +38,56 @@
   end
 
   describe '#yell_at_patrons' do
-    it 'returns a list of uppercased names' do
-      skip
+    it 'returns a list of uppercased patron names' do
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+
+
+  # Iteration 3
+
+  describe '#over_capacity?' do
+    it 'returns false because venue is not over capacity' do
```

Comment:
Nice job testing the `false` here. You could combine this test with your next test by adding the 3 patrons, then have your expectation for false, then below that add your other 2 patrons and a final expectation for true